server.js

```diff
@@ -0,0 +1,39 @@
+console.log('this is my server');
+
+const express = require('express');
+const app = express();
+
+require('dotenv').config();
+
+const PORT = process.env.PORT;
+
+const weatherDate = require('./data/weather.json');
+
+
+app.get('/', (request, response) => {
+    response.send('hello from the server!');
+});
+
+app.get('/weather', (request, response) => {
+    console.log(request.query);
+    const {lat, lon, searchQuery}=request.query
```

Comment:
I don't know if this works the way I think it does, but if it does it's very cool.